Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -115,6 +115,7 @@ class PlaceDetailsCardView: UIView {
     lazy var tripAdvisorReviewView: ReviewContainerView = {
         let view = ReviewContainerView(color: Colors.tripAdvisor, mode: .detailsView)
         view.reviewSiteLogo.image = UIImage(named: "logo_ta")
+        view.isUserInteractionEnabled = true
```

Comment:
no, actually this should not be on this view at all. I think I removed it from the label and left it here when it should have been the other way around